assets/app/views/createProject.html

```diff
@@ -34,19 +34,19 @@
         </div>
 
         <div class="form-group">
-          <label for="displayName">Display Name</label>
+          <label for="openshift.io/displayName">Display Name</label>
```

Comment:
don't need to change the html field names, only where we actually read the annotation off the API object